Code/Framework/AzFramework/Platform/Linux/platform_nativeui_linux.cmake

```
@@ -18,13 +18,16 @@ if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
     set(LY_FILES_CMAKE
         Platform/Common/Xcb/azframework_xcb_files.cmake
     )
+    include(Platform/Common/Xcb/generate_splash_screen_header.cmake)
     set(LY_BUILD_DEPENDENCIES
         PRIVATE
             3rdParty::X11::xcb
             3rdParty::X11::xcb_xkb
             3rdParty::X11::xcb_xfixes
             3rdParty::X11::xkbcommon
             3rdParty::X11::xkbcommon_X11
+            xcb_image
```

I had to apply this to compile it. With that works as expected.

```suggestion
            xcb-image
```